src/components/VLink.vue

```diff
@@ -1,27 +1,21 @@
 <!-- eslint-disable vue/no-restricted-syntax -->
 <template>
   <NuxtLink
-    v-if="linkComponent === 'NuxtLink'"
-    v-bind="$attrs"
+    v-if="isNuxtLink"
     :class="{ 'inline-flex flex-row items-center gap-2': showExternalIcon }"
     :to="linkTo"
     v-on="$listeners"
     @click.native="$emit('click', $event)"
   >
-    <slot /><VIcon
-      v-if="showExternalIcon && !isInternal"
-      :icon-path="externalLinkIcon"
-      class="inline-block"
-      :size="4"
-      rtl-flip
-    />
+    <slot />
```

Comment:
I realized just now that a NuxtLink is only used for the internal links, so it cannot use the `externalLinkIcon`.

Reply:
Interesting. How do we define an "external" link? When we've removed the iframe, will w.org links be "external"? If all links that don't use `NuxtLink` are external then we can remove the conditional `showExternalIcon && !isInternal` from `VIcon`. If it's only things that are not either (a) Openverse frontend links or (b) w.org links, then we could detect it and automatically add the icon without having to rely on the prop.

Probably an additional tightening up here and clarification around the iframe project and the idea of "externality". We typically concern ourselves with this (I think) due to differing privacy expectations on other websites. Given Openverse and WordPress.org follow the same privacy policies and are all controlled by WPF, we probably don't need the external icon for w.org links either. But again, something to clarify and probably something for another issue and to rope the w.org infrastructure folks in on.